.prettierrc.json

```diff
@@ -0,0 +1,7 @@
+{
+  "semi": false,
+  "singleQuote": true,
+  "useTabs": false,
+  "tabWidth": 2,
+  "printWidth": 120
```

Comment:
you may want to remove this file, but then prettier in vscode/intellij will format with defaults